libkodimote/episodes.cpp

```diff
@@ -179,9 +165,9 @@ void Episodes::listReceived(const QVariantMap &rsp)
         list.append(item);
         m_idIndexMapping.insert(item->episodeId(), index++);
     }
-    beginInsertRows(QModelIndex(), 0, list.count() - 1);
+    beginResetModel();
```

Comment:
Thinking about it, that's probably a bit more work to handle all the cases where remote items could have been added/removed in the meantime. Let's do that in another branch.